cars.csv

```diff
@@ -0,0 +1 @@
+ Id : 51 Color : blue, Id : 42 Color : green, Id : 83 Color : red, Id : 14 Color : pink, Id : 65 Color : gray, Id : 86 Color : red, Id : 17 Color : black, Id : 48 Color : white, Id : 99 Color : black, Id : 10 Color : blue
```

Comment:
Make more readable e.g. multi-lines instead of using one line. It's hard to read.